LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2020 Harris Hancock
```

Comment:
+ @chreekat, now?

Reply:
meh, eventually, or "The contributors". Not sure what best practice is.